File: NavigationJavaSample/app/src/main/java/com/example/navigationjavasample/ui/detail/DetailFragment.java
package com.example.navigationjavasample.ui.detail;

import android.os.Bundle;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.navigationjavasample.Person;
import com.example.navigationjavasample.R;
import com.example.navigationjavasample.databinding.FragmentDetailBinding;

public class DetailFragment extends Fragment {
    private FragmentDetailBinding binding;

    public DetailFragment() {
        super(R.layout.fragment_detail);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        binding = FragmentDetailBinding.bind(view);
        final Person person = DetailFragmentArgs.fromBundle(requireArguments()).getPerson();
        binding.textName.setText("名前： "+ person.getName());
        binding.textAge.setText("年齢： " + person.getAge());
        binding.textHeight.setText("身長： " +  String.format("%.1f", person.getHeight()));
    }
}

File: NavigationKotlinSample/app/src/main/java/com/example/navigationkotlinsample/ui/detail/DetailFragment.kt
package com.example.navigationkotlinsample.ui.detail

import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import androidx.navigation.fragment.navArgs
import com.example.navigationkotlinsample.Person
import com.example.navigationkotlinsample.R
import com.example.navigationkotlinsample.databinding.FragmentDetailBinding

class DetailFragment : Fragment(R.layout.fragment_detail) {
    private var binding: FragmentDetailBinding? = null
    private val navArgs by navArgs<DetailFragmentArgs>()
    override fun onViewCreated(
        view: View,
        savedInstanceState: Bundle?
    ) {
        super.onViewCreated(view, savedInstanceState)
        binding = FragmentDetailBinding.bind(view)
        val person: Person = navArgs.person
        binding?.also {
            it.textName.text = "Name： " + person.name
            it.textAge.text = "Age： " + person.age
            it.textHeight.text = "Height： " + String.format("%.1f", person.height)
        }
    }
}